incubator/nodejs/image/project/package.json

```diff
@@ -11,6 +11,6 @@
   "dependencies": {
   },
   "devDependencies": {
-    "appmetrics-dash": "^4.1.0"
+    "appmetrics-dash": "^5.0.0"
```

Comment:
Can we update the version of stacks stack.yaml for all 3 node stacks?

Reply:
We might want to keep package.json in sync too

Reply:
Thank you, I have done this in the most recent commit